### DISCUSSION TITLE
z2m installation on OH3

### DISCUSSION THREAD

**Discussion by TorbenMei** [Koenkk/zigbee2mqtt#12320] [General]
Hi
Just made a new SD-card openHAB 3.2.0 - under "optional componentets" i installed mosquitto - but "Zigbee2mqtt" is not accessible - any ideas?
Torben

**Comment by brushcutter** [comment]
Hi TorbenMei,
I'm using pi4 with openhabian 3.2.0 release - there is no optional component in this release for zigbee2mqtt mentioned in the install instructions that I've seen. I was able to get the zigbee2mqtt running by following the linux install instructions, however I've yet to be able to get an Ember Coordinator to work with it using the Sonoff Zigbee USB 3.0 Dongle.

Anyone have the SONOFF Zigbee 3.0 USB Dongle Plus working with Openhab release 3.2.0? I'd love to try your settings! Everything I've been trying to get z2m to use it has resulted in either a Comm Error or "Unknown" state and I've never seen any devices show up in the z2m frontend. After a few days of trying, I ordered a Nortek adaptor and was able to get that functioning in Openhab, but z2m doesn't support that adaptor (yet).

Good Luck!

**Reply by brushcutter** [reply]
I should also mention: I purchased two SONOFF Zigbee 3.0 USB Dongle Plus and updated one of them using these instructions:
https://sonoff.tech/wp-content/uploads/2021/12/SONOFF-Zigbee-3.0-USB-dongle-plus-firmware-flashing-1-1.pdf
I'm now searching how to get the firmware revision from the adaptor to understand which one I have working. I've read it's supposed to work "out of the box", but I've yet to be able to configure an Ember Coordinator with either.
